packages/hoprd/hopr-admin/src/utils/command.ts

```diff
@@ -159,66 +225,87 @@ export const CMD_PARAMS: Record<CmdTypes, CmdArg<any, any, any>> = {
     }
   ],
   nativeAddress: [
-    'A native address',
+    'NATIVE address',
+    "'0x..",
     (v) => {
       return [ethersUtils.isAddress(v), v]
     }
   ],
   hoprAddressOrAlias: [
-    'A HOPR address (PeerId) or an alias',
+    'HOPR address or alias',
+    "'16Ui..' or 'alice'",
     (peerIdStrOrAlias, { aliases }) => {
       // is PeerId
       let peerId: PeerId | undefined
 
       // try PeerId
       try {
         peerId = peerIdFromString(peerIdStrOrAlias)
-      } catch {
-        console.log(`Could not create peer id from '${peerIdStrOrAlias}'`)
-      }
+      } catch {}
 
       // try aliases
       if (!peerId && aliases) {
         const alias = aliases[peerIdStrOrAlias]
         try {
           peerId = peerIdFromString(alias)
-        } catch {
-          console.log(`Could not create peer id from alias '${alias}' for '${peerIdStrOrAlias}'`)
-        }
+        } catch {}
       }
 
       if (peerId) return [true, peerId]
       return [false]
     }
   ] as CmdArg<string, { aliases: Record<string, string> }, PeerId>,
   hoprOrNative: [
-    "'HOPR' or 'NATIVE'",
+    'currency',
+    "'hopr' or 'native'",
+    (input) => {
+      if (typeof input !== 'string') return [false, input]
+      const v = input.toLowerCase()
+      return [v === 'hopr' || v === 'native', v]
+    }
+  ],
+  direction: [
+    'direction',
+    "'incoming' or 'outgoing'",
+    (input) => {
+      if (typeof input !== 'string') return [false, input]
+      const v = input.toLowerCase()
+      return [v === 'incoming' || v === 'outgoing', v]
+    }
+  ],
+  constant: [
+    'constant',
+    'A constant value',
     (v) => {
-      return [v === 'HOPR' || v === 'NATIVE', v]
+      return [true, v]
     }
   ],
   number: [
-    'A number',
+    'number',
+    'Any number',
     (v) => {
       return [!isNaN(v), v]
```

Comment:
Please use `isFinite` since `isNaN(Infinity) -> false` and things such as `Infinity + Infinity = Infinity`

Reply:
fixed in 8800e19581364d583b9f31077159c520bc323d51